gamemodes/terrortown/entities/entities/ttt_base_placeable.lua

```diff
@@ -392,4 +407,58 @@ if SERVER then
 
         return true
     end
-end
+end -- SERVER
+
+if CLIENT then
+    local TryT = LANG.TryTranslation
+    local ParT = LANG.GetParamTranslation
+
+    function ENT:ShouldDrawMarkerVision()
+        return true
+    end
+
+    function ENT:CustomizeMarkerVision(mvData) end
+
+    function ENT:GetMarkerVisionIcons(mvData)
+        return { self.iconMaterial }
+    end
+
+    function ENT:GetMarkerVisionColors(mvData)
+        return { COLOR_WHITE }
```

Comment:
Why is this a color table? Is this one color for each icon? One for each line of text?

Reply:
Ah, I see this is for the icon, this is a bit confusing, because the text can have colors as well

Reply:
Good to know. Will adjust.